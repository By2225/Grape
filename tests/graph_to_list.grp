fun Int main() {
    Graph<Int, Int> a = << '7' -8- '9'>>;
    List<Node<Int > > b = graph_to_list(a);
    return 0;
}
