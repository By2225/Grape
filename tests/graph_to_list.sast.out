
fun Int main() {
Graph<Node<Int>, Edge<Int->Int>> (Graph<Node<Int>, Edge<Int->Int>> : a = (Graph<Node<Int>, Edge<Int->*>> : <(Node<Int> : '(Int : 7)') (Edge<Int->*> : -(Int : 8)-) (Node<Int> : '(Int : 9)') (Void : )>));
List<Node<Int>> (List<Node<Int>> : b = (List<Node<Int>> : graph_to_list((Graph<Node<Int>, Edge<Int->Int>> : a))));
return (Int : 0);
}
